### ISSUE TITLE
[Feature Request] New Snout Cape Design

### ISSUE THREAD

**Issue by Slumsy** [sp614x/optifine#5441]
**TL;DR** - No snout cape design on banner generator site, could implement for variation

I'm not sure if this is something that isn't possible to implement but it would be neat to have the new snout pattern supported for capes. I don't think this is currently a thing. Please correct me if I'm wrong, I've tried looking for some way to implement it but the banner generator doesn't have the option for a snout banner. 

It has this option on the Planet Minecraft banner editor but, if there is some way to manually edit the banner code to implement the snout, please inform me and feel free to close this issue. If not, I hope it isn't asking for too much by adding the snout as a supported pattern for the capes, it would definitely include a bit more versatility.

Thanks in advance for reading this issue.

**Comment by Cart3r1234** [comment]
This is something that Needscoolshoes needs to handle rather than something optifine can handle (unless optifine were to switch banner sites, which is unlikely).  If I'm not mistaken, NCS has asked for permission from Mojang to use the new patterns on their site, but simply has never heard back from them.
